es_loader/es_loader.py

```diff
@@ -41,12 +72,49 @@ def _create_index(self):
         self.es.indices.create(self.index_name)
         logger.debug('Index created')
 
+    def _get_job(self):
+        """Get job from ScrapingHubClient istance."""
+        self.job = self.sc.get_job(self.job_id)
+
+    def _get_job_metadata(self):
+        """Get job metadata."""
+        self.metadata = self.job.metadata.list()
+
+    def _get_scraped_items_count(self):
+        """Get scraped items count from job metadata.
+
+        The count is needed to make log messages more verbose
+        and to calculate batches size"""
+        self._get_job_metadata()
+        for data in self.metadata:
+            if data[0] == 'scrapystats':
+                self.items_count = data[1]['item_scraped_count']
+
     def _get_items(self):
-        logger.debug('Getting items')
-        items = self.sc.get_job(self.job_id).items.iter()
+        """Get items from Scrapy Cloud.
+
+        Returns:
+            items - an iterator with items.
+        """
+        logger.debug(
+            'There are {} items scraped. Downloading...'.format(self.items_count)
+        )
+        items = self.job.items.iter()
```

Comment:
I would also just inline this method for clarity.
When I am looking at `process_items`, it is impossible to immediately see what the item is in the `for item in self._get_items()`.
While if I see something like `for item in self.job.items.iter()`, I get a better idea of what is happening.